src/gap_filler.py

```diff
@@ -1,43 +1,65 @@
+import math
 import random
 import spacy
-import utils
+from src import utils
+
 
 class GapFinder:
```

Comment:
Also we need to ensure all methods of the GapFinder class return data to the calling function as we require this for testing :)

Reply:
Got it, re: returning data. No problem - I haven't forgotten, I was just working on that one function (find_gaps) with my latest updates.